### PR TITLE
Add deploy script

### PR DESCRIPTION
**Pull request by koudaiii**
## WHY

ci push image to quay.io/wantedly/slack-notification.
## WHAT
- set deploy script
- set travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,17 +1,30 @@
-sudo: required
-services:
-  - docker
 language: go
 go:
-  - '1.7'
+- '1.7'
+services:
+- docker
+addons:
```
Comment (dtan4) [inline .travis.yml]: Is this really required?
`glide` should be installed by `make deps`, I think.

**Review by dtan4 — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 1 above)
Comment (koudaiii) [reply .travis.yml]: This commit mistake.
I will delete addons.

Comment (koudaiii) [reply .travis.yml]: ooops.
I discover new mistake..
I will fix Makefile and glide.

**Comment by koudaiii** [comment]
image

**Comment by koudaiii** [comment]
Please review 🙇

**Review by dtan4 — APPROVED** [review]

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**